backend/src/test/java/gov/cdc/usds/simplereport/api/TranslatorTest.java

```diff
@@ -0,0 +1,245 @@
+package gov.cdc.usds.simplereport.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import static gov.cdc.usds.simplereport.api.Translators.parseEmail;
+import static gov.cdc.usds.simplereport.api.Translators.parseEthnicity;
+import static gov.cdc.usds.simplereport.api.Translators.parseGender;
+import static gov.cdc.usds.simplereport.api.Translators.parseRace;
+import static gov.cdc.usds.simplereport.api.Translators.parseState;
+import static gov.cdc.usds.simplereport.api.Translators.parseString;
+import static gov.cdc.usds.simplereport.api.Translators.parseUserShortDate;
+import static gov.cdc.usds.simplereport.api.Translators.parseUUID;
+import static gov.cdc.usds.simplereport.api.Translators.parseYesNo;
+
+import java.time.LocalDate;
+
+import org.junit.jupiter.api.Test;
+
+import gov.cdc.usds.simplereport.api.model.errors.IllegalGraphqlArgumentException;
+
+@SuppressWarnings("checkstyle:MagicNumber")
+public class TranslatorTest {
+    @Test
+    public void testEmptyShortDate() {
+        assertEquals(null, parseUserShortDate(""));
```

Comment:
nice test

Reply:
Although having seen this in an alert I now feel compelled to point out that `assertNull` is probably more appropriate here. 😁